docs/user/filesystem/architecture.rst

```diff
@@ -0,0 +1,70 @@
+FileSystem Datastore Architecture
+=================================
+
+The GeoMesa FileSystem Datastore (GeoMesa FSDS) takes advantage of the performance characteristics of modern
+cloud-native and distributed filesystems to scale bulk analytic queries. The FSDS is a good choice for doing bulk egress
+queries or large analytic jobs using frameworks such as Spark SQL and MapReduce. The FSDS differs from other queries
```

Comment:
This sentence " The FSDS differs from other queries in that ingest latency is traded for high-throughput query" seems weird.  The FSDS is not a query, right?